preprocessing/text/tokenizer.py

```diff
@@ -0,0 +1,64 @@
+from phonemizer.phonemize import phonemize
+
+from preprocessing.text.symbols import _phonemes, _punctuations
+from preprocessing.text.cleaners import English, German
+
+
+class Pipeline:
```

Comment:
Thats a good module. It would be better though to inject the dependencies. https://en.wikipedia.org/wiki/Dependency_injection#:~:text=In%20software%20engineering%2C%20dependency%20injection,object%20is%20called%20a%20service. 

This will be important if we use a class like this in production as we need to be flexible in designing the tokenizer and cleaner.